tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/BrowserBased/BrowserAdditionalInformation/Supplier Edit BrowserAdditionalInformation Validation.feature

```diff
@@ -10,7 +10,7 @@ Background:
     And Solutions exist
         | SolutionId | SolutionName   | SupplierStatusId | SupplierId |
         | Sln1       | MedicOnline    | 1                | Sup 1      |
-    And SolutionDetail exist
+    And Solution have following details
```

Comment:
I think this might read better if it was `And solutions have the following details`.

As this will affect most of the files in the PR I'll wait until you've updated this before continuing the review.

Reply:
Updated feature steps with `solutions have the following details`